rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -222,6 +222,19 @@ usb_handler_thread::usb_handler_thread()
 			found_skylander = true;
 		}
 
+		if (desc.idVendor == 0x1209 && desc.idProduct == 0x2882)
+		{
+			sys_usbd.success("Found device: Santroller");
+			libusb_device_handle* lusb_handle;
+			libusb_open(list[index], &lusb_handle);
+#ifdef __linux__
+			libusb_set_auto_detach_kernel_driver(lusb_handle, true);
+			libusb_claim_interface(lusb_handle, 2);
+#endif
+			libusb_control_transfer(lusb_handle, +LIBUSB_ENDPOINT_IN | +LIBUSB_REQUEST_TYPE_CLASS | +LIBUSB_RECIPIENT_INTERFACE, 0x01, 0x03f2, 2, nullptr, 0, 5000);
+			libusb_close(lusb_handle);
```

Comment:
There is no comment explaining what you're doing here. So anyone reading this has no idea what's happening.